packages/ui/src/components/Drawer/Drawer.tsx

```diff
@@ -33,6 +33,12 @@ export interface DrawerProps extends _DrawerProps {
    * Title that appears at the top of the drawer
    */
   title: string;
+  /**
+   * Adds a suffix element to the drawer.
+   *
+   * If set, the drawer will feature a suffix element in the title.
```

Comment:
```suggestion
   * Can be used to indicate special states, such as `BetaChip`, `NewFeatureChip`, or other UI elements next to the title.
```

I think it would be better to document this optional prop to make its usage clearer.